include/mbedtls/havege.h

```diff
@@ -43,9 +44,9 @@ extern "C" {
  */
 typedef struct mbedtls_havege_state
 {
-    int PT1, PT2, offset[2];
-    int pool[MBEDTLS_HAVEGE_COLLECT_SIZE];
-    int WALK[8192];
+    uint32_t PT1, PT2, offset[2];
```

Comment:
How about the local variables in the `havege_fill` function? The shift operations are performed on them rather than these structure members.

Reply:
Done in https://github.com/ARMmbed/mbed-crypto/pull/149/files#diff-d31fd2e88e9f9dc591a6b54765880556L159

Building without the crypto submodule isn't supported anymore, is it? So only changes to header files need to be duplicated between mbed-crypto and mbedtls, right?